app/src/main/java/org/simple/clinic/summary/bloodsugar/BloodSugarSummaryViewModel.kt

```diff
@@ -3,18 +3,20 @@ package org.simple.clinic.summary.bloodsugar
 import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
 import org.simple.clinic.bloodsugar.BloodSugarMeasurement
+import org.simple.clinic.facility.Facility
 import java.util.UUID
 
 @Parcelize
 data class BloodSugarSummaryViewModel(
     val patientUuid: UUID,
     val measurements: List<BloodSugarMeasurement>?,
-    val totalRecordedBloodSugarCount: Int?
+    val totalRecordedBloodSugarCount: Int?,
+    val currentFacility: Facility?
```

Comment:
I tried doing this first. Let's discuss this. This was not working as expected.
Also, it would still need me to keep the `FetchCurrentFacility` effect. Right?

Reply:
It won't be necessary since we can query the facility just before invoking the method on the UI action.